CREATE USER [py_etl];

